package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@react-ui-org/react-ui",
   "description": "React UI is a themeable UI library for React apps.",
-  "version": "0.43.0",
+  "version": "0.44.0-alpha.0",
```

Maybe `0.44.0-alpha.1`?

---

[SemVer](https://semver.org/#spec-item-9) only speaks of `-alpha` or `-alpha.1`, `-alpha.2`etc. But it's no big deal for me.

---

E.g. https://github.com/npm/node-semver#prerelease-identifiers. I've also read articles about is as it should end with `.0`

---

I wanted to discuss it on Monday's meeating but It did not happen.
```suggestion
  "version": "0.44.0-alpha.0",
```